packages/relay-runtime/store/OperationExecutor.js

```diff
@@ -250,13 +252,15 @@ class Executor<TMutation: MutationParameters> {
       !RelayFeatureFlags.PROCESS_OPTIMISTIC_UPDATE_BEFORE_SUBSCRIPTION &&
       optimisticConfig != null
     ) {
-      this._processOptimisticResponse(
-        optimisticConfig.response != null
-          ? {data: optimisticConfig.response}
-          : null,
-        optimisticConfig.updater,
-        false,
-      );
+      this._schedule(() => {
```

Comment:
I've noticed we don't have tests which validate this code path (all tests still pass with this wrapper removed)

Reply:
Ah right looks like the feature flag is set in an earlier test run and not restored https://github.com/facebook/relay/blob/56ad111455344980f8481798b92a25d2049ba725/packages/relay-runtime/store/__tests__/RelayModernEnvironment-ExecuteMutation-test.js#L556.
I'll update the new tests to run with all relevant feature flag states.